presto-docs/src/main/sphinx/security/kerberos-principals.fragment

```diff
@@ -0,0 +1,29 @@
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Figured it out. I was going to try to pull this section out into a shared fragment, but the content is different enough that it didn't make sense to do so. This file is dead code; I'll remove it.